playbooks/roles/xqwatcher/tasks/deploy_watcher.yml

```diff
@@ -15,7 +15,7 @@
     - install:app-requirements
 
 - name: "Install Datadog APM requirements"
-  when: COMMON_ENABLE_DATADOG and COMMON_ENABLE_DATADOG_APP
+  when: XQWATCHER_DATADOG_ENABLED
```

Comment:
Some instances in this file are `ENABLED` instead of `ENABLE`.

Reply:
oh whew thank you for catching this 🤦